docs/source/tasks/intermediateCA.rst

```diff
@@ -5,7 +5,7 @@
 Creating an Intermediate Certificate Authority
 =================================================
 
-IBM Blockchain Platform allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The IBM Blockchain Platform documentation describes the business rerquirement and the manual steps in detail.
+IBM Support for Hyperledger Fabric allows for the optional use of Intermediate CAs which can protect a root CA and provide operational flexibility. The IBM Support for Hyperledger Fabric documentation describes the business rerquirement and the manual steps in detail.
```

Comment:
Similar thing here, need to generalise the wording since this applies to the Fabric OSS as well

Reply:
Also might want to get rid of the reference to the documentation since this is a Hyperledger Fabric repo